gui/wxpython/rlisetup/wizard.py

```diff
@@ -1230,11 +1230,8 @@ def afterRegionDrawn(self, marea):
             self.parent.wizard.ShowPage(self.parent.summarypage)
         else:
             self.title.SetLabel(
-                _(
-                    "Draw sample region "
-                    + str(self.regioncount)
-                    + " of "
-                    + str(numregions)
+                _("Draw sample region {count} of {num_regions}").format(
+                    count=self.regioncount, num_regions=numregions
```

Comment:
Does python need more than this to format numbers correctly, or the default string representation is enough?